PnX-SI/GeoNature
# Feat/add destination front

Content:
- Added destinations selection to the modal in the frontend
- Moved dataset selection from the modal to the Upload page
- Added the destination to the routing of the frontend
- Added the destination code in the Import fetched
- Translated a few things

To test:
- Do an import in different destinations and see if it works
- Exit the import at differents steps and test
